docs/en/source/callFlow.rst

```diff
@@ -145,27 +138,29 @@ The Figure is showing the operational flow of context source registration in the
 
 2. The request enters in service API gateway.
 
- a. The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
+ 2.1 The service API gateway discovers the actual serving micro-service endpoints (where the incoming requests need to be forwarded) from discovery & registry service.
 
- b.The service API gateway forwards the HTTP request to the Context Registry (CR) Manager micro-service.   
+ 2.2 The service API gateway forwards the HTTP request to the Context Source Registry micro-service.
 
-3. The CR manager now fetches the previously stored data/entities from the Topic “CSource”.
+3. The Context Source Registry now fetches the previously stored data/entities from the Topic “CSource”.
 
- a. If the entry for the request csource is already present it exits the processing and informing the same to the requester. If it is not present, then it continues for further processing.
+- If the entry for the request csource is already present it exits the processing and informing the same to the requester. If it is not present, then it continues for further processing.
 
- b.Now the CR manager performs some basic validation to check if this is a valid request with the valid payload. 
+- Now the Context Source Registry performs some basic validation to check if this is a valid request with the valid payload.
 
- c.CR manager now writes this payload into the Csoure Topic.
+- Context Source Registry now writes this payload into the CSource Topic.
 
-4. The Storage Manager will keep listening for the Csource topic and for any new entry write it perform the relative operation in the database. 
+4. Now, our csource registration request firstly gets publish/store in the database and after that further processing is done in the Kafka bus.
 
-5. The CR manager prepares the response for csource request and
+5. Context Source Registry will keep listening to the CSource topic and for any new entry.
 
- 5.1 sends the Http response back to the API gateway.
+6. Now, Context Source registry prepares the response for csource registration request.
 
- 5.2 API gateway sends back the response to the end-user/requester.
+ 6.1 sends the Http response back to the API gateway.
 
-**Note**: For Conext Source Update request only the payload will get changes and in step 3 upon validation for the existing entity it will not exit rather it will update the retrieved entity and write it back into the Kafka. The rest of the flow will remain mostly the same. 
+ 6.2 API gateway sends back the response to the end-user/requester.
+
+**Note**: For Conext Source Update request only the payload will get changes and in step 3 upon validation for the existing entity it will not exit rather it will update the retrieved entity and write it back into the Kafka. The rest of the flow will remain mostly the same.
```

Comment:
[Major] Please changes in flow diagram 4
- Remove CSource and CSource sub topics direction 
-  Registry topic direction same as above
-   Add from Registry topic direction to Registry Subscription manager
-   Changes in theoretical part as well